src/test/rpc_wallet_tests.cpp

```diff
@@ -1520,21 +1520,21 @@ BOOST_AUTO_TEST_CASE(rpc_z_shieldcoinbase_parameters)
     std::string mainnetzaddr = "zcMuhvq8sEkHALuSU2i4NbNQxshSAYrpCExec45ZjtivYPbuiFPwk6WHy4SvsbeZ4siy1WheuRGjtaJmoD1J8bFqNXhsG6U";
 
     try {
-        std::shared_ptr<AsyncRPCOperation> operation(new AsyncRPCOperation_shieldcoinbase(mtx, {}, testnetzaddr, -1 ));
+        std::shared_ptr<AsyncRPCOperation> operation(new AsyncRPCOperation_shieldcoinbase(TransactionBuilder(), mtx, {}, testnetzaddr, -1 ));
```

Comment:
This seems to be improving coverage for Sapling shielded transactions at the expense of coverage for Sprout shielded transactions :-( Ideally we'd test both.

Reply:
No, this isn't the same mechanism as with `z_sendmany` (where the presence of `TransactionBuilder` indicates Sapling usage). Here, we only have a shielded output address, so we can simply run the Sprout-or-Sapling logic based on the output address type. The changes here are just to satisfy the new constructor argument requirements.